css/style.css

```diff
@@ -0,0 +1,486 @@
+}
+
+/* comments form */
+.comments-form {
+  flex-direction: row;
+  padding-top: 20px;
+}
+
+.comment-form-icon {
+  width: 10%;
+}
+
+.comment-form-icon > img {
+  border-radius: 50%;
+}
+
+.comments-text-box {
+  width: 85%;
+}
+
+.comments-text-box > input,
+.comments-text-box > input:hover,
+.comments-text-box > input:focus {
+  width: 100%;
+  border: none;
+  border-bottom: 1px solid grey;
+  background: transparent;
+  outline: none;
+}
+
+/* comments buttons */
+.comment-buttons {
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.comment-buttons > .cancel-btn {
+  padding: 10px 20px;
+  text-transform: uppercase;
+  border: none;
+  font-weight: 500;
+}
+
+.comment-buttons > .comment-btn {
+  padding: 10px 20px;
+  text-transform: uppercase;
+  border: none;
+  border-radius: 5%;
+  font-weight: 500;
+}
+
+/* comments card */
+.comments-card {
+  flex-direction: row;
+}
+
+.comment-card-icon {
+  width: 7%;
+}
+
+.comment-card-icon > img {
+  border-radius: 50%;
+}
+
+.comment-card-column {
+  width: 85%;
+  flex-direction: column;
+}
+
+.name-time-frame {
+  flex-direction: row;
+}
+
+.name-time-frame > .commenter {
+  font-weight: bold;
+  padding: 0 3px;
+  margin: 2px 0;
+}
+
+.name-time-frame > .comment-time {
+  padding: 0 3px;
+  margin: 2px 0;
+}
+
+.comment-thumbs {
+  flex-direction: row;
+  padding: 10px;
+  padding-left: 0;
+}
+
+.comment-thumbs > i {
+  padding: 0 7px;
+}
+
+.comment-thumbs > div {
+  padding: 0 15px;
+}
+
+.comment-replies {
+  flex-direction: row;
+}
+
+/* side bar */
+.side-videos {
+  flex-direction: column;
+  width: 30%;
+  margin: 20px 20px;
+}
+
+.embeded-video {
+  width: 100%;
+  height: 400px;
+}
+
+.thumbnail-card {
+  flex-direction: row;
+  padding: 10px 0;
+  cursor: pointer;
+  flex-wrap: wrap;
+}
+
+.thumbnail-card > .thumbnail-image {
+  width: 40%;
+  height: 94px;
+  padding-right: 15px;
+}
+
+.thumbnail-card > .thumbnail-image > img {
+  max-width: 100%;
+  height: auto;
+}
+
+.thumbnail-card > .thumbnail-title {
+  width: 50%;
+  margin-top: 0;
+  padding: 0;
+  height: 94px;
+}
+
+.thumbnail-card > .thumbnail-title > * {
+  margin: 0;
+}
+
+.thumbnail-card > .thumbnail-title > p {
+  padding: 5px;
+}
+
+/*  sidebar autoplay */
+.side-autoplay {
+  flex-direction: row;
+
+}
+
+.side-autoplay > .side-upnext {
+  width: 30%;
+}
+
+.side-autoplay > .side-upnext > p {
+  font-weight: bold;
+  padding: 10px;
+  font-size: 1rem;
+  color: #302d2d;
+}
+
+.side-autoplay > .side-toggle {
+  flex: 1;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+.side-autoplay > .side-toggle > div {
+  align-self: center;
+  align-items: center;
+}
+
+.side-autoplay > .side-toggle > div > .autoplay-text {
+  text-transform: uppercase;
+  margin-right: 10px;
+  font-weight:bolder;
+}
+
+.side-autoplay > .side-toggle > div > i {
+  font-size: 1.5rem;
+}
+
+.next-play {
+  border-bottom: 1px solid rgb(199, 195, 195);
+}
+
+/* .main-video {
+            display: flex;
+        } */
```

Comment:
[OPTIONAL] It's optional since it wasn't pointed earlier, but I strongly suggest you guys to implement it.
- [ ] Comments about the code are allowed, commented code not. Please take care of it boys 💪